### PR TITLE
Fix PhpDoc in yii\db\Query

### PR DESCRIPTION
**Pull request by WinterSilence**
Fix PhpDoc's for methods with parameter `$db = null`

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | -

### CODE REVIEW

File: framework/db/Query.php [hunk 1]
```diff
@@ -475,11 +476,8 @@ protected function queryScalar($selectExpression, $db)
             $this->limit = null;
             $this->offset = null;
 
-            $e = null;
             try {
                 $command = $this->createCommand($db);
-            } catch (\Exception $e) {
```
Comment (samdark) [inline framework/db/Query.php:481]: That would break things on older PHP versions where there's no Throwable.

File: framework/db/Query.php [hunk 2]
```diff
@@ -489,7 +487,7 @@ protected function queryScalar($selectExpression, $db)
             $this->limit = $limit;
             $this->offset = $offset;
 
-            if ($e !== null) {
+            if (isset($e)) {
```
Comment (samdark) [inline framework/db/Query.php:490]: That's not a good practice overall to rely on abscense of a variable.

**Review by samdark — CHANGES_REQUESTED** [review]

File: framework/db/Query.php (hunk 2 above)
Comment (WinterSilence) [reply framework/db/Query.php:490]: @samdark yes, but this often used in current code
https://github.com/yiisoft/yii2/blob/6b1f70b883cc091644a30529d7d106c954abd1ce/framework/db/ActiveRelationTrait.php#L202 
https://github.com/yiisoft/yii2/blob/6b1f70b883cc091644a30529d7d106c954abd1ce/framework/db/ActiveRelationTrait.php#L272 
that's confused me... what's better: write code in "yii style" or based on good practices?

File: framework/db/Query.php (hunk 1 above)
Comment (WinterSilence) [reply framework/db/Query.php:481]: reverted

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks.

**Pull request merged**